compiler/passes/convert-uast.cpp

```diff
@@ -1104,7 +1104,9 @@ struct Converter {
 
   /// StringLikeLiterals ///
   Expr* visit(const uast::BytesLiteral* node) {
-    std::string quoted = quoteStringForC(node->str().toString());
```

Comment:
`std::string quoted = quoteStringForC(node->str().str());`
(similar changes needed for other diffs in this file)